backend/fms_core/template_importer/importers/sample_update.py

```diff
@@ -0,0 +1,57 @@
+from fms_core.models import Process, Protocol
+
+from ._generic import GenericImporter
+from fms_core.template_importer.row_handlers.sample_update import SampleRowHandler
+
+class SampleSubmissionImporter(GenericImporter):
+    SHEETS_INFO = [
+        {'name': 'SampleUpdate', 'header_row_nb': 4},
```

Comment:
This will cause troubles eventually people sometime remove lines or add lines. Ideally we would Check for a marker in the file instead. I realized for exemple that if i have one less line in a one line template it will tell me that my operation is successful (because there is supposedly no entry ... This is false and could lead to missing data in the system.
The most obvious marker is the list of headers. If it does match it is the header ... We could also use another marker but it risk being removed edited.